puzzle/cli/cli.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+import os
+import logging
+
+import click
+
+from codecs import open
+from puzzle.log import configure_stream, LEVELS
+
+from puzzle.settings import BaseConfig
+from puzzle.plugins import SqlStore, VcfPlugin
+
+from . import (root, verbose)
+
+logger = logging.getLogger(__name__)
+
+
+@click.group()
+@root
+@verbose
+@click.pass_context
+def cli(ctx, verbose, root):
```

Comment:
Wondering if this is a little ridiculous to type `from puzzle.cli.cli import cli` hehe

Might I suggest: `puzzle/cli/root/root_cmd` or something similar?